E2ETests/Runner/Scenarios/OpenTelemetryTrace/OpenTelemetryTraceView.swift

```diff
@@ -0,0 +1,99 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import SwiftUI
+import OpenTelemetryApi
+
+class DistributedTraceDelegate: NSObject, URLSessionDataDelegate {
+}
+
+class OpenTelemetryTraceViewController: UIHostingController<OpenTelemetryTraceView> {
+    init(tracer: OpenTelemetryApi.Tracer, urlSession: URLSession) {
+        super.init(rootView: OpenTelemetryTraceView(tracer: tracer, urlSession: urlSession))
+    }
+
+    @MainActor required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+struct OpenTelemetryTraceView: View {
+
+    let tracer: OpenTelemetryApi.Tracer
+    let urlSession: URLSession
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                 Form {
+                     Section(header: Text("Local span, starts and ends in the same thread")) {
+                         Button(action: {
+                             tracer.spanBuilder(spanName: "Local Span").startSpan().end()
+                         }) {
+                             Text("Generate")
+                         }
+                     }
+
+                     Section(header: Text("Distributed span, calling httpbin.org")) {
```

Comment:
Are the spans automatically created for this section?

Reply:
Yes, because of https://github.com/DataDog/dd-sdk-ios/blob/ganeshnj/tests/trace-e2e/E2ETests/Runner/Scenarios/OpenTelemetryTrace/OpenTelemetryTraceScenario.swift#L43-L47

We are instrumenting all the requests made using this client on httpbin.org host.